app/connectors/ApplicationClient.scala

```diff
@@ -228,6 +228,12 @@ trait ApplicationClient {
       case ex: Throwable => throw new ErrorRetrievingAvailableSchemes(ex)
     }
   }
+
+  def getAvailableSchemes(implicit hc: HeaderCarrier): Future[List[SchemeInfo]] = {
+    http.GET(s"$hostBase/available/schemes").map { response =>
```

Comment:
If you change the route in backend, as I suggested or something better, you will have to change this one too.